### PR TITLE
Add login page content, change app to load onto login page with login button currently directing to dashboard view.

### PR DESCRIPTION
**Pull request by roonil-wazlib**

### CODE REVIEW

File: portfolio/frontend/src/containers/Login/Login.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+            <div className={classes.Body}>
+                <div className={classes.Column}>
+                    <h1>Who's the next trendsetter?</h1>
+                    <p>Bacon ipsum dolor amet ground round chislic bresaola, andouille pig buffalo swine t-bone strip steak. 
+                        Kevin shoulder frankfurter, venison short loin t-bone shank chislic chuck beef ribs pork buffalo. 
+                        Rump hamburger jowl cow. Tail ribeye burgdoggen tongue. Chislic pork belly doner rump frankfurter. 
+                        Jowl meatloaf ribeye beef pork, shoulder bresaola filet mignon.</p>
+                    <div className={classes.Buttons}>
+                        <button className={classes.Login}>Login</button>
```
Comment (lin8231) [inline portfolio/frontend/src/containers/Login/Login.js:23]: Might wanna add the `onClick={() => props.handler()` here as well so this button redirects tooo

Comment (roonil-wazlib) [reply portfolio/frontend/src/containers/Login/Login.js:23]: Oh yeah...

Comment (roonil-wazlib) [reply portfolio/frontend/src/containers/Login/Login.js]: Ok, fixed that. I'm failing the javascript styling check on the code you gave me, but I'm guessing that's for the reasons you mentioned about it being bad and temporary?

Comment (texm) [reply portfolio/frontend/src/containers/Login/Login.js]: The styling checks are purely cosmetic - `make pretty`should make them go away now that `clang-format` is working.

Comment (roonil-wazlib) [reply portfolio/frontend/src/containers/Login/Login.js]: make pretty introduces a number of errors for me currently, it's doing some weird things like replacing some > with '.

Comment (roonil-wazlib) [reply portfolio/frontend/src/containers/Login/Login.js]: I suppose i can just keep the changes it makes on this file for now.

**Comment by lin8231** [comment]
LGTM

**Review by lin8231 — APPROVED** [review]

File: portfolio/frontend/src/containers/Login/Login.js (hunk 1 above)
Comment (roonil-wazlib) [reply portfolio/frontend/src/containers/Login/Login.js]: Ok I've done that and pushed the style changes.

**Review by texm — APPROVED** [review]

**Pull request merged**